src/Operations.jl

```diff
@@ -992,12 +992,12 @@ function build_versions(ctx::Context, uuids::Set{UUID}; verbose=false)
         local build_project_override, build_project_preferences
         if isfile(projectfile_path(builddir(source_path)))
             build_project_override = nothing
-            with_load_path([builddir(source_path)]) do
+            with_load_path([Base.LOAD_PATH..., builddir(source_path)]) do
```

Comment:
Is the order here right? I would have expected the LOAD_PATH to come after not before

Reply:
Yeah, I think you're right.